src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -432,6 +446,26 @@ public List<String> run() {
         }
     }
 
+    private void verifyTLSVersion(final String settings, final List<String> configuredProtocols) {
+        for (final var tls : configuredProtocols) {
+            if (tls.equalsIgnoreCase("TLSv1")) {
+                deprecationLogger.deprecate(
+                    settings,
+                    "The '{}' setting contains TLSv1 protocol version which was deprecated since 2021 (RFC 8996). "
+                        + "Support for it will be removed in the next major release.",
+                    settings
+                );
+            } else if (tls.equalsIgnoreCase("TLSv1.1")) {
+                deprecationLogger.deprecate(
+                    settings,
+                    "The '{}' setting contains TLSv1.1 protocol version which was deprecated since 2021 (RFC 8996). "
+                        + "Support for it will be removed in the next major release.",
+                    settings
+                );
+            }
```

Comment:
```suggestion
            if (tls.equalsIgnoreCase("TLSv1") || tls.equalsIgnoreCase("TLSv1.1") ) {
                deprecationLogger.deprecate(
                    settings,
                    "The '{}' setting contains '{}' protocol version which was deprecated since 2021 (RFC 8996). "
                        + "Support for it will be removed in the next major release.",
                    settings, tls
                );
            }
             
```

Reply:
done